webapp/components/user_settings/custom_theme_chooser.jsx

```diff
@@ -156,17 +156,35 @@ class CustomThemeChooser extends React.Component {
         this.props.updateTheme(theme);
     }
 
+    getColors(text) {
+        const colorsText = text.split(', ');
```

Comment:
what happens if the string does not have spaces before or after the `,` can you make sure to handle that case